test/boilerplate_suggestors/stubbed_props_and_state_class_remover_test.dart

```diff
@@ -12,24 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'dart:convert';
-
 import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
 import 'package:over_react_codemod/src/boilerplate_suggestors/stubbed_props_and_state_class_remover.dart';
 import 'package:test/test.dart';
 
 import '../util.dart';
-import 'boilerplate_utilities_test.dart';
 
 main() {
   group('StubbedPropsAndStateClassRemover', () {
-    final testSuggestor = getSuggestorTester(
-      StubbedPropsAndStateClassRemover(),
-    );
-
-    setUpAll(() {
-      semverHelper = SemverHelper(jsonDecode(reportJson));
-    });
+    final semverHelper = getSemverHelper(
```

Comment:
This will be removed when #82 (which removes the need for `SemverHelper` in `StubbedPropsAndStateClassRemover`) is merged in.